asv_bench/benchmarks/series_methods.py

```diff
@@ -135,7 +135,7 @@ def time_isin_long_series_long_values_floats(self):
 
 class IsInLongSeriesLookUpDominates:
     params = [
-        ["int64", "int32", "float64", "float32", "object", "Int64", "Float64"],
+        ["int64", "int32", "float64", "float32", "object"],
```

Comment:
Can you instead raise a NotImplementedError in the `setup` below if `dtype in ("Int64", "Float64") and np.__version >+ "1.20.0"` ?

That keeps the benchmark running for if you don't have numpy 1.20 installed (although I don't know how often the environment gets updated on our benchmark server)

Reply:
Thanks @jorisvandenbossche . done.

Reply:
Thanks!

Further follow-up question: 1) is there actually an issue about this failure? 2) can you add a link to that issue as a `TODO` comment where raising the NotImplementedError?

Reply:
> is there actually an issue about this failure?

#39844



> can you add a link to that issue as a `TODO` comment where raising the NotImplementedError?

as a follow-up